guides/InstallationAdminGuide.md

```diff
@@ -539,7 +565,20 @@ Where:
 * ‘nocanon’ ensures URLs are passed through to the host without processing.
 
 An example directive is:
-![alt text](ExampleDirectiveSSL.png "Example Directive SSL")
+
+```apache
+<VirtualHost *:443>
+  ServerName www.equella.com
```

Comment:
Might as well change these to the generic form as well

```
ServerName {external-server-name}
ProxyPass / http://{equella-host}:{http-port}/ nocanon
```

Reply:
So duplicate the example on line 547?

Reply:
Updated :ballot_box_with_check:

Reply:
yeah